src/renderers/shared/shared/event/BrowserEventConstants.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule BrowserEventConstants
+ */
+
+'use strict';
+
+var getVendorPrefixedEventName = require('getVendorPrefixedEventName');
+
+export type PropagationPhases = 'bubbled' | 'captured';
```

Comment:
Ah right, it gets referenced here: https://github.com/facebook/react/pull/9512/files#diff-1ca53ffa4da75a64f7c3b2f131f62da7R21